RELEASE/scripts/autoscend/quests/level_12.ash

```diff
@@ -312,6 +312,10 @@ WarPlan auto_bestWarPlan()
 	{
 		considerArena = false;
 	}
+	if(get_property("auto_skipNuns").to_boolean())
+	{
+		considerNuns = false;
+	}
```

Comment:
`WarPlan auto_bestWarPlan()` should respect the property "auto_skipNuns".
since it has been added here, special dispensation for that property in `boolean L12_clearBattlefield()` is no longer necessary